packages/react-instantsearch/src/core/createInstantSearchManager.derived.test.js

```diff
@@ -76,46 +61,46 @@ describe('createInstantSearchManager', () => {
 
         expect(ism.store.getState().results).toBe(null);
 
-        return Promise.resolve().then(() => {
-          jest.runAllTimers();
-
-          const store = ism.store.getState();
-          expect(store.results.first).toEqual({
-            query: 'first query 1',
-            page: 3,
-            index: 'first',
-          });
-          expect(store.results.second).toEqual({
-            query: 'second query 1',
-            page: 0,
-            index: 'second',
-          });
-          expect(store.error).toBe(null);
-
-          ism.widgetsManager.getWidgets()[0].getSearchParameters = params =>
-            params.setQuery('first query 2');
-          ism.widgetsManager.getWidgets()[1].getSearchParameters = params =>
-            params.setQuery('second query 2');
-
-          ism.widgetsManager.update();
-
-          return Promise.resolve().then(() => {
-            jest.runAllTimers();
-
-            const store1 = ism.store.getState();
+        return Promise.resolve()
+          .then(() => {})
```

Comment:
I wonder if “await await” is allowed? I think a comment here would be useful too

Reply:
Agree for comment!